files/zh-cn/learn/tools_and_testing/understanding_client-side_tools/command_line/index.md

```diff
@@ -284,7 +286,7 @@ curl https://developer.mozilla.org/docs/Web/API/fetch -L -I | grep location
 输出应该是这样的 (`curl` 首先会输出一些下载计数器之类的东西):
 
 ```bash
-location: /en-US/docs/Web/API/fetch
+location: /zh-CN/docs/Web/API/fetch
```

Comment:
The location given in the example is the actual behavior of the server:

![image](https://github.com/mdn/translated-content/assets/15844309/caa63f90-8a91-48c6-95fb-3c695c12cc4f)